docs/docs/manual/exporting.md

```diff
@@ -6,34 +6,115 @@ sidebar_label: Exporting
+
+On the <span class="tabLabels">Content</span> tab, you can drag and drop the columns appearing in the column list to reorder the output. The options for reconciled and date data are applied to each column individually. 
+
+This exporter is especially useful with reconciled data, as you can choose whether you wish to output the cells' original values, the matched values, or the matched IDs. Ouputting “match entity's name”, “matched entity's ID”, or “cell's content” will output, respectively, the contents of `cell.recon.match.name`, `cell.recon.match.id`, and `cell.value`. 
+
+“Output nothing for unmatched cells” will export empty cells for both newly-created matches and cells with no chosen matches. “Link to matched entity's page” will produce hyperlinked text in an HTML table output, but have no effect in other formats.
+
+At this time, the date-formatting options in this window do not work. You can [keep track of this issue on Github](https://github.com/OpenRefine/OpenRefine/issues/3368).
+In the future, you will also be able to choose how to [output date-formatted cells](exploring#dates). You can create a custom date output by using [formatting according to the SimpleDateFormat parsing key found here](grelfunctions#todateo-b-monthfirst-s-format1-s-format2-).
+
+![A screenshot of the custom tabular file download tab.](/img/custom-tabular-exporter2.png)
+
+On the <span class="tabLabels">Download</span> tab, you can generate a preview of how the first ten rows of your dataset will output. If you do not choose one of the file formats on the right, the <span class="buttonLabels">Download</span> button will generate a text file. On the <span class="tabLabels">Upload</span> tab, you can create a new Google Sheet. 
+
+With the <span class="tabLabels">Option Code</span> tab, you can copy JSON of your current settings to reuse on another project, or you can paste in existing JSON settings to apply to the current project. 
+
+### SQL exporter
+
+The SQL exporter creates a SQL statement containing the data you’ve exported, which you can use to overwrite or add to an existing database. A window with two tabs will pop up: one to define what data to output, and another to modify the statement table. 
+
+![A screenshot of the SQL statement creator window.](/img/sql-exporter.png)
+
+The <span class="tabLabels">Content</span> tab allows you to craft a complete SQL statement, including DROP and IF EXISTS if you require them.
+
+You can set a default value for empty cells after unchecking “Allow null” in one or more columns. Trimming column names will remove their whitespace characters. 
 
-*   Custom tabular export
-*   SQL
-*   Templating export \
+![A screenshot of the SQL statement download window.](/img/sql-exporter2.png)
 
-*   Upload edits to Wikidata
-*   Export to QuickStatement
-*   Export Wikidata schema
+The <span class="tabLabels">Download</span> tab allows you to finalize your statement, name it, and preview or download it.
```

Comment:
"tab allows you to finalize your statement, name it, and preview or download it."
This suggests that you can "name your statement" but in face you can set a name for the table which the statement will refer to